src/NServiceBus.Core/DetectObsoleteConfigurationSettings.cs

```diff
@@ -43,6 +44,19 @@ static void DetectObsoleteConfiguration(UnicastBusConfig unicastBusConfig)
             }
         }
 
+        static void DetectObsoleteConfiguration(AuditConfig auditConfig)
+        {
+            if (!string.IsNullOrWhiteSpace(auditConfig?.QueueName))
+            {
+                Logger.Warn($"The use of the {nameof(AuditConfig.QueueName)} attribute in the {nameof(AuditConfig)} configuration section is discouraged and will be removed in the next major version. Remove this from the configuration section. Switch to the code API by using `{nameof(EndpointConfiguration)}.AuditProcessedMessagesTo` instead.");
```

Comment:
why do you have the statement `Remove this from the configuration section.` on the queue name but not on OverrideTimeToBeReceived ? It reads a bit harsh to me, maybe we should just leave this sentence out?